packages/types/src/userMemory/tools.ts

```diff
@@ -0,0 +1,241 @@
+import { z } from 'zod';
+
+import { UserMemoryContext, UserMemoryExperience, UserMemoryPreference } from '@/database/schemas';
```

Comment:
Types should not rely on the schema here; they need to be defined separately and repeatedly